sdks/python/apache_beam/examples/inference/pytorch_image_classification_with_side_inputs.py

```diff
@@ -26,8 +26,11 @@
 This pipeline follows the pattern from
 https://beam.apache.org/documentation/patterns/side-inputs/
 
-This pipeline expects a PubSub topic as source, which emits an image
-path(UTF-8 encoded) that is accessible by the pipeline.
+To use the PubSub reading from a topic in the pipeline as source, you can
+publish a path to the model(resnet152 used in the pipeline from
+torchvision.models.resnet152) to the PubSub topic. Then pass that
+topic via command line arg --topic.  The published path(str) should be
+UTF-8 encoded.
```

Comment:
Looks like docs precommit doesn't like some of your indentation here

Reply:
Yes, corrected it